### PR TITLE
Feature/feed list

### PR DESCRIPTION
**Pull request by griev04**
In this PR, navigation (switch, tab and stack) was added along with basic views.
Feed list still needs some work.
Mock data is now being used for the feed list (that should be fetched from the backend, not from github).

### CODE REVIEW

File: views/FeedPost.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  handleListTap(item, userAction='details') {
+    this.props.navigation.navigate("Post", {
+      displayText: buildPost(item),
+      item: item,
+      userAction: userAction,
+    });
+  }
+
+  render() {
+    const displayText = buildPost(this.props.item);
```
Comment (enima2684) [inline views/FeedPost.js]: - [x]  change item name to 'event'

File: views/FeedPost.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      } at ${item.created_at}`;
+      break;
+
+    case "ForkEvent":
+      displayText = `${item.actor.login} forked ${item.repo.name} of ${
+        item.org.login
+      } at ${item.created_at}`;
+      break;
+    case "PushEvent":
+      displayText = `${item.actor.login} commited ${item.payload.before} to ${
```
Comment (enima2684) [inline views/FeedPost.js]: - [x] Truncate commit id to 7 chars

File: views/FeedPost.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      break;
+
+    default:
+      displayText = `${item.actor.login} did something else: ${item.type}`;
+      break;
+  }
+  return displayText;
+}
+
+export default class FeedScreen extends React.Component {
```
Comment (enima2684) [inline views/FeedPost.js]: - [x] CHange class name to FeedPost

File: views/FeedPost.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  
+  /**
+   * 
+   * @param {*} item 
+   * @param {*} userAction user clicks on post or on comment button. Possible values: details, comment
+   */
+  handleListTap(item, userAction='details') {
```
Comment (enima2684) [inline views/FeedPost.js]: - [x] Remove this part for the moment

**Comment by enima2684** [comment]
- [ ] Props on components

**Comment by enima2684** [comment]
- [ ] proptypes on components

File: views/HomeScreen.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { StyleSheet, View, Button } from "react-native";
+
+export default class HomeScreen extends React.Component {
```
Comment (enima2684) [inline views/HomeScreen.js]: - [x] Remove the file

- [x] Rename classes to their right names

**Review by enima2684 — CHANGES_REQUESTED** [review]

File: views/FeedPost.js (hunk 4 above)
Comment (griev04) [reply views/FeedPost.js]: This code is okay, the one breaking was in PostScreen: now it's removed

File: views/FeedPost.js (hunk 1 above)
Comment (griev04) [reply views/FeedPost.js]: changed to feedEvent, event is too generic imho

**Review by enima2684 — APPROVED** [review]

**Pull request merged**